package.json

```diff
@@ -86,7 +86,7 @@
   },
   "devDependencies": {
     "@axe-core/react": "^4.6.1",
-    "@pact-foundation/pact": "12.1.1",
+    "@pact-foundation/pact": "13.2.0",
```

Comment:
So this would solve:
- https://github.com/DataBiosphere/terra-ui/security/dependabot/172
- https://github.com/DataBiosphere/terra-ui/security/dependabot/170

And https://github.com/DataBiosphere/terra-ui/security/dependabot/169 simply cannot be patched at the moment?

Reply:
Yes, and it also closes https://github.com/DataBiosphere/terra-ui/security/dependabot/173, a sneaky duplicate `path-to-regexp` alert that opened since this ticket was created. I added a note to AN-312 about the `validate.js` alert.